bigquery/samples/tests/test_load_table_dataframe.py

```diff
@@ -25,7 +28,67 @@ def test_load_table_dataframe(capsys, client, random_table_id):
 
     table = load_table_dataframe.load_table_dataframe(client, random_table_id)
     out, _ = capsys.readouterr()
-    assert "Loaded 4 rows and 3 columns" in out
+    expected_column_names = [
+        "wikidata_id",
+        "title",
+        "release_year",
+        "length_minutes",
+        "release_date",
+        "dvd_release",
+    ]
+    assert "Loaded 4 rows and {} columns".format(len(expected_column_names)) in out
 
     column_names = [field.name for field in table.schema]
-    assert column_names == ["wikidata_id", "title", "release_year"]
+    assert column_names == expected_column_names
+    column_types = [field.field_type for field in table.schema]
+    assert column_types == [
+        "STRING",
+        "STRING",
+        "INTEGER",
+        "FLOAT",
+        "TIMESTAMP",
+        # Due to internal bug 147108331, BigQuery always interprets DATETIME
+        # columns as having the wrong precision. In the meantime, workaround this
+        # by writing the values as TIMESTAMP. See:
+        # https://github.com/googleapis/google-cloud-python/issues/9996
+        "TIMESTAMP",
+    ]
+
+    df = client.list_rows(table).to_dataframe()
```

Comment:
The name `client` is not defined here.

Reply:
Ah yes, I removed those in my mass change to uncomment the client constructor in samples. Fixed by adding `client` to test function's fixtures.